mozilla_bitbar_devicepool/configuration.py

```diff
@@ -81,8 +99,13 @@ def configure(bitbar_configpath, filespath=None, update_bitbar=False):
 
     with open(bitbar_configpath) as bitbar_configfile:
         CONFIG = yaml.load(bitbar_configfile.read(), Loader=yaml.SafeLoader)
-    expand_configuration()
     logger.info('configure: performing checks')
+    try:
+        ensure_filenames_are_unique(CONFIG)
+    except (ConfigurationFileException, ConfigurationFileDuplicateFilenamesException) as e:
```

Comment:
This is fine but with the change to make the two exceptions a hierarchy we could get by with just ConfigurationFileException.

Reply:
fixed.